android/src/main/java/io/mosip/tuvali/wallet/transfer/message/InitTransferMessage.kt

```diff
@@ -1,3 +1,3 @@
 package io.mosip.tuvali.wallet.transfer.message
 
-class InitResponseTransferMessage(val data: ByteArray): IMessage(TransferMessageTypes.INIT_RESPONSE_TRANSFER)
+class InitResponseTransferMessage(val data: ByteArray,val mtuSize: Int): IMessage(TransferMessageTypes.INIT_RESPONSE_TRANSFER)
```

Comment:
(nitpick) As we've seen Android SDK's MTU includes the ATT Headers of MTU and iOS does not, should we tweak the parameter names to `maxChunkSize` or `maxDataBytes` instead of MTU ? 

In Android's terminology, we are passing maximum chunk size but we the parameters say that we are passing MTU, this is a small term-jumble but it should be confusing for us after we've long forgotten Bluetooth concepts and individual platform nuances.

Reply:
I see that the param name is mtuSize, but when we are using this we are setting it to `mtuSize - HEADERS_SIZE_IN_MTU`